### PR TITLE
Fix image paper feed

### PR DESCRIPTION
**Pull request by ma-zal**
Fixes issue grandchef/escpos-buffer#46.

### CODE REVIEW

**Comment by ma-zal** [comment]
Important note: This change breaks 5 tests. So please double check if all of these model outputs should be changed, or if previous implementation for some of them was correct.

Changes output of:
- `__tests__/resources/cmp-20_qrcode.bin`
- `__tests__/resources/dr800_draw_buffer.bin`
- `__tests__/resources/mp-4200_th_draw_buffer.bin`
- `__tests__/resources/mp-4200_th_draw_file.bin`
- `__tests__/resources/printid_qrcode.bin`

It this is correct, I will update all these files for all tests passing (by additional commit). Should be done before PR approval. In other cases, further discussion is needed. :-)

**Comment by mazinsw** [comment]
Thanks for this pull request fix, i will test on printer this new binaries, but its problably that this image printing will get spacing between two lines for model MP-4200 TH, i will put the results here

**Comment by ma-zal** [comment]
Thanks for the answer @mazinsw. I am interested in testing your model.
I found [the specifications of your MP-4200 TH](https://www.bztech.com.br/arquivos/manual-programacao-bematech-mp-4200-th.pdf).

There is the interesting message on page 34:

> A full print line with ‘N’ columns will need N*3 bytes. If the image to print has less than ‘N’ graphic
columns, a LF command must be sent to perform line printing. Text and graphic may be mixed
in the same line.

So for a smaller image, it looks like the `0x0A` is correct. But for sure we can also test if a behavior is the same on images with the exact maximum width.

**Review by samueldurantes — APPROVED** [review]

File: src/profile/index.ts [hunk 1]
```diff
@@ -126,15 +126,20 @@ export abstract class Profile {
...
     for (let y = 0; y < image.lines; y++) {
+      // Bit-image slice
       const data = image.lineData(y);
       await this.connection.write(
         Buffer.from(this.bitmapCmd + low + high, 'ascii'),
       );
       await this.connection.write(data);
-      await this.connection.write(Buffer.from('\x1BJ\x00', 'ascii'));
+      // Paper feed
+      await this.connection.write(Buffer.from('\x0A', 'ascii'));
```
[the remark this answers is not in the document]
Comment (ma-zal) [reply src/profile/index.ts:140]: @sibelius Yes, only this line one. ... All others are just code comments improvements.

**Comment by ma-zal** [comment]
Dear @samueldurantes.
I just rebased my PR to help to merge. Please merge this PR ... or else close without merging if not wanted. Thanks.

**Comment by ma-zal** [comment]
@samueldurantes or @grandchef - Should I close this PR? It is already reviewed, and approved, but nobody merged it.

**Comment by samueldurantes** [comment]
@mazinsw can I merge this PR?

**Comment by mazinsw** [comment]
I have not tested yet (sorry i forgot), i remember that this line break and create a horizontal spacing between image lines on some of these affected printers, if you can test and merge, do it, because i'am out of this repo/company
